framework/src/st_things/things_stack/things_stack.c

```diff
@@ -807,7 +803,8 @@ OCEntityHandlerResult things_abort(pthread_t *h_thread_abort, things_es_enrollee
 	}
 
 	if ((*h_thread_abort) == 0) {
-		if ((ARGs = (s_abort_s *) things_malloc(sizeof(s_abort_s))) == NULL) {
+		s_abort_s *ARGs = (s_abort_s *) things_malloc(sizeof(s_abort_s));
```

Comment:
remove space between "(s_abort_s *)" and things_mallo

Reply:
Okay.

Note: This code is removed in https://github.com/Samsung/TizenRT/pull/2535